services/frontend-service/src/ui/utils/Links.tsx

```diff
@@ -116,41 +118,5 @@ export const DisplayLink: React.FC<{ displayString: string; app: string; version
             </a>
         );
     }
-    return <span>{displayString}</span>;
-};
-
-export const ReleaseVersionLink: React.FC<{
-    displayVersion: string;
-    undeployVersion: boolean;
-    sourceCommitId: string;
-    version: number;
-    app: string;
-}> = (props): JSX.Element => {
-    const { displayVersion, undeployVersion, sourceCommitId, version, app } = props;
-    if (undeployVersion) {
-        return <span>Undeploy Version</span>;
-    }
-    if (displayVersion !== '') {
-        return (
-            <span>
-                Release Version: <DisplayLink displayString={displayVersion} app={app} version={String(version)} />
-            </span>
-        );
-    }
-    if (sourceCommitId !== '') {
-        return (
-            <span>
-                CommitID: <DisplayLink displayString={sourceCommitId} app={app} version={String(version)} />
-            </span>
-        );
-    }
-    if (version > 0) {
-        return (
-            <span>
-                Version: <DisplayLink displayString={String(version)} app={app} version={String(version)} />
-            </span>
-        );
-    }
-
-    return <span></span>;
+    return null;
```

Comment:
I thought the idea was to bring this ReleaseVersionLink back.
If you don't do it in your PR, then I can do it, but you have to mention it clearly in the Commit Message.
I would not want to release this as it is now, because it removes a feature.

Reply:
It doens't remove the feature at all. The link is still present as a text link below the commit message. See here: https://github.com/freiheit-com/kuberpult/pull/947/files#diff-010deeee4eee8f461d029a4411c49b374fa144d3c04a9eb8f21398d984bd0d51R302

Reply:
Ahh, ok!